teams/roles-and-responsibilities.md

```diff
@@ -1 +1,65 @@
+    - with the community in an exemplary way - for better or for worse, the “core team” is often perceived as a unit, not as individuals
+    - with other members of the core teams in a respectful way (do we need to say this?)
+- Lead the official projects through some combination of
+    - Triaging issues
+    - Reviewing PRs and mentoring community contributors
+    - Answering community questions (Discord, [Discuss](https://discuss.emberjs.com/), Stack Overflow, etc)
+    - Supporting community documentation efforts
+    - Writing blog posts for emberjs.com
+    - Championing RFCs
+    - Supporting the release process
+- Attend weekly team meetings
+- Be reliable
+    - If you say you can do something, do it
+    - If you can’t do it (life happens- and we are all volunteers) let the team know
+
+
+## Annual Commitments
+
+The expectations for participation is that every core team member is expected to participate, in the way(s) that they define for themselves. 
+
+Each team member should define the ways in which they will participate, and will revisit this on an annual basis. This will help to ensure that the core teams continue to challenge ourselves and grow. It will also help us identify what type of core team member we are missing, so we can help ourselves grow a balanced team. 
+
+There are many different ways to participate as a core team member:
+
+- Writing new feature code
+- Refactoring old code to improve it
+- Writing documentation
+- Technical Mentorship
+    - Regularly answering questions in chat & on the forum
+    - Training new potential maintainers - working to “grow” the next group of Ember core team members
+- Speaking about Ember at Conferences and Meetups
+- Writing RFCs
+- Leading projects (i.e., writing the quest issue, providing project management, hosting strike team calls)
+
+Note: “thought leadership” is not a form of participation. Every core team member is expected to already be capable of this before joining the core team (whether they choose to voice thought leadership or not).  
+
+Lack of participation (either by not defining goals, or not honoring the commitments made) is one of the indicators that alumnization should be considered. However, if goals have been defined, but a team member has a major life event, a sabbatical may be considered. 
+
+## Sabbatical
+
+There are certain types of life events- illness of a family member, moving to a new state/country, starting a new job, even burnout- that may require a core team member to take a brief leave of absence from their team. 
+
+The team member should inform their team and the steering committee of this absence, as well as the length of the absence, which could last up to 3 months. 
+
+(How do we indicate that someone is on sabbatical?) 
+
+(Steps to take a sabbatical)
+
+There are some life events that require a longer, or permanent absence from the core team- in those instances, alumnization should be considered instead.  
+
+## Alumnization
```

Comment:
We had previously decided that the correct term for people here is [emeritus](https://en.wikipedia.org/wiki/Emeritus)

> The term emeritus does not necessarily signify that a person has relinquished all the duties of their former position, and they may continue to exercise some of them.

Whereas alumnization infers nothing about retaining the abilities of a core team member (but none of the obligations).

Is this an intentional shift in thinking?

Reply:
That's a good question.

Reply:
I am still a believer in "emeritus" and think we should still use that term.

Reply:
Yes, same. I'd love to see those concepts incorporated. IMHO, the emeritus line of thinking is fundamentally required in order for folks to be even a tiny bit comfortable with moving off of a core team...

Reply:
ok, I will update this wording. I noticed that the website was probably where I pulled the phrase so I also have submitted a PR to fix it there too.